svc-events/evmodel/events.go

```diff
@@ -566,3 +593,45 @@ func DeleteUndeliveredEventsFlag(destination string) error {
 	}
 	return nil
 }
+
+// SaveAggregateSubscription is to save subscription details of device
+func SaveAggregateSubscription(aggregateID string, hostIP []string) error {
```

Comment:
As discussed this should be done in Aggregation Service during the creation of aggregate, not during event subscription please   update the same

Reply:
done